classes/01class/exercises/c01-git03/mail2kvanitha/README.md

```diff
@@ -0,0 +1,23 @@
+# c01-git03
+
+## Questions (answer inline)
+
+1. Did you notice any difference in the size of the repo before and after adding the big file?
+The size of the repo increased significantly after adding the big file
+
+2. What is the reason for this problem?
+Git maintains the history of commits of each file/branches, and hence the repo size grows with each commit.
+
+3. How do you remove something from Git history after it is pushed to the remote repository? Which commands would you use? 
+"git filter-branch --index-filter "git rm -rf --cached --ignore-unmatch path_to_file HEAD" to remove file from local repository
```

Comment:
That works too, but the new recommended tool for this is `git filter-repo`. It is simpler to use and safer.